docs/wiki/framework/refuel-framework.md

```diff
@@ -16,8 +16,15 @@ version:
 
 ```cpp
 class CfgVehicles {
-    class MyFuturisticMBT {
-        ace_refuel_fuelCapacity = 3000; // set to correct path static model
+    class MyFuelTruck {
+        ace_refuel_fuelCargo = 3000; // Fuel cargo
+        ace_refuel_hooks)[] = {{0.38,-3.17,-.7},{-0.41,-3.17,-.7}}; // Nozzle hooks positions
```

Comment:
it seams there is a ) to much -> ace_refuel_hooks**)**[]

Reply:
will fix in 1 day